src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -330,6 +343,9 @@
                 <attribute name="denormalization_context">
                     <attribute name="groups">admin:cart:update</attribute>
```

Comment:
Not related to changes in this PR, but it looks like it should be `shop`, not `admin` here.